R/package.R

```diff
@@ -6,40 +6,75 @@
 #' @importFrom pkgdepends remotes
 #' @importFrom pkginstall install_packages
 #' @importFrom crayon blue
+#' @importFrom glue backtick glue
 #' @export
 pkg_install <- function(pkg, lib = .libPaths()[[1L]], num_workers = 1L) {
   r <- remotes$new(pkg, library = lib)
 
   # Solve the dependency graph
   r$solve()
+  sol <- r$get_solution()
+
+  # Solution error?
+  if (sol$status != "OK") {
+    msg <- paste(format(sol$failures), collapse = "\n")
+    cat("\n")
```

Comment:
Why do we need an extra newline here? Because the pkgdepends progress bar does not add one when resolution completes?

Reply:
Because the error block might be several lines, and it is good to separate it from the rest of the output.